core/domain/services/graphql/mutators/EntityReorder.php

```diff
@@ -142,26 +147,26 @@ public static function prepareEntityDetailsFromInput($input)
 
         $entities = $model::instance()->get_all([
             [
-                $primaryKey => ['IN', $entityDbids],
+                $primaryKey => ['IN', $entity_db_ids],
                 $deletedKey => ['IN', [true, false]],
             ],
         ]);
 
         /**
          * If we could not get exactly same number of entities for the given DB IDs
          */
-        if (count($entityDbids) !== count($entities)) {
+        if (count($entity_db_ids) !== count($entities)) {
             throw new UserError(esc_html__('Sorry, operation cancelled due to missing entities.', 'event_espresso'));
         }
 
         // Make sure we have an instance for every ID.
-        foreach ($entityDbids as $entityDbid) {
-            if (isset($entities[ $entityDbid ]) && $entities[ $entityDbid ] instanceof EE_Base_Class) {
+        foreach ($entity_db_ids as $entity_db_id) {
+            if (isset($entities[ $entity_db_id ]) && $entities[ $entity_db_id ] instanceof EE_Base_Class) {
                 continue;
             }
             throw new UserError(esc_html__('Sorry, operation cancelled due to invalid entities.', 'event_espresso'));
         }
 
-        return compact('entities', 'entityGuids', 'entityDbids', 'entityType', 'keyPrefix');
+        return compact('entities', 'entityGuids', 'entity_db_ids', 'entityType', 'keyPrefix');
```

Comment:
This was changed without updating the consumers of this array.

Reply:
i bet that was a find n replace error...
i was most likely changing a variable name from `pascalCase` to `snake_case` but was searching specifically for `$entityDbids` 

sry about that